sample-authz/src/main/java/net/trajano/ms/example/authz/TokenResource.java

```diff
@@ -0,0 +1,177 @@
+package net.trajano.ms.example.authz;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.JWSAlgorithm;
+import com.nimbusds.jose.jwk.source.RemoteJWKSet;
+import com.nimbusds.jose.proc.BadJOSEException;
+import com.nimbusds.jose.proc.DefaultJOSEProcessor;
+import com.nimbusds.jose.proc.JWSVerificationKeySelector;
+import com.nimbusds.jose.proc.SimpleSecurityContext;
+import com.nimbusds.jwt.JWTClaimsSet;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiParam;
+import net.trajano.ms.common.beans.TokenGenerator;
+import net.trajano.ms.common.oauth.ClientValidator;
+import net.trajano.ms.common.oauth.GrantTypes;
+import net.trajano.ms.common.oauth.IdTokenResponse;
+import net.trajano.ms.common.oauth.OAuthTokenResponse;
+import net.trajano.ms.example.authz.sample.Clients;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.security.PermitAll;
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.*;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.UriBuilderException;
+import java.io.IOException;
+import java.net.URI;
+import java.text.ParseException;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Date;
+
+@Api
+@Configuration
+@Component
+@Path("/token")
+@PermitAll
+public class TokenResource {
+
+    private static final Logger LOG = LoggerFactory.getLogger(TokenResource.class);
+
+    @Autowired
+    private TokenGenerator tokenGenerator;
+
+    @Autowired
+    private ClientValidator clientValidator;
+
+    @Value("${realmName:client_credentials}")
+    private String realmName;
+
+    private Clients clients;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "clients" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)